src/main/java/de/tum/cit/aet/artemis/iris/web/open/PublicPyrisStatusUpdateResource.java

```diff
@@ -126,7 +128,7 @@ public ResponseEntity<Void> setCompetencyExtractionJobStatus(@PathVariable Strin
     }
 
     /**
-     * {@code POST /api/public/pyris/pipelines/text-exercise-chat/runs/{runId}/status} : Set the status of a Text Exercise Chat job.
+     * {@code POST /api/public/pyris/pipelines/lecture-chat/runs/{runId}/status} : Set the status of a Lecture Chat job.
```

Comment:
You changed the description of the wrong method imo - the `responseInTextExerciseChat` should still be about `text-exercises` not lectures, right? I suggest reverting those changes

Reply:
Good point. Should be changed now